Update SoCreate.ServiceFabric.PubSub.csproj

WhitWaldo opened the pull request:
Updated to Service Fabric SDK 6.0.1121

waynemunro commented:
There are also some updated nuget packages available. Please would you update them as well?
Also please update the changelog file accordingly. Thanks.

WhitWaldo commented:
@waynemunro Updated to 15.1.0 and updated changelog. Didn't see any newer NuGet packages for the main project, but updated the SF package on the example and updated the SF Mocks package on the tests to reflect a PR of mine that was accepted this morning as well (same SF version increment).

waynemunro commented:
> @WhitWaldo thank you. I've approved what I can. I don't have write access.
